ci/docker/install/ubuntu_julia.sh

```diff
@@ -32,7 +32,7 @@ function install_julia() {
     # The julia version in Ubuntu repo is too old
     # We download the tarball from the official link:
     #   https://julialang.org/downloads/
-    wget -qO $JLBINARY https://s3.amazonaws.com/julialang2/bin/linux/x64/0.7/julia-0.7.0-linux-x86_64.tar.gz
```

Comment:
The problem is not about the S3 path but that you hardcode the 0.7 version instead of using `$1` and `$2`. In any case, hopefully https://julialang-s3.julialang.org/bin/linux/x64/$1/julia-$2-linux-x86_64.tar.gz will remain stable for the future

Reply:
Thanks for pointing it out. I'll replace it back to the S3 path.